server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -685,6 +685,14 @@ public boolean isES87TSDBCodecEnabled() {
         Property.Final
     );
 
+    public static final FeatureFlag DOC_VALUES_SPARSE_INDEX = new FeatureFlag("doc_values_sparse_index");
+    public static final Setting<Boolean> USE_DOC_VALUES_SPARSE_INDEX = Setting.boolSetting(
+        "index.mapping.use_doc_values_sparse_index",
```

Comment:
Let's use the lucene name. So `use_doc_values_skipper`?

Reply:
Note that in `IndexVersions` I think I cannot rename `public static final IndexVersion HOSTNAME_DOC_VALUES_SPARSE_INDEX = def(9_008_0_00, Version.LUCENE_10_0_0);`
I will rename everything else.